js/apps.js

```diff
@@ -9,6 +9,21 @@ class Book {
     this.author = author;
     this.id = null;
   }
+
+  createBook = (title, author) => {
+    bookArray.push({ title, author });
+  }
+
```

Comment:
- [ **OPTIONAL** ]  I think it would be better if you add an `id` property in the **Book** object as right now your book is added with only **title** and **author** properties.